src/components/CustomText/utils.tsx

```diff
@@ -0,0 +1,207 @@
+import { TextStyle } from "react-native";
+
+export const VariantTypesPossible = [
+  "headingLarge",
+  "headingNormal",
+  "headingSmall",
+  "heading",
+  "subHeadingLarge",
+  "subHeadingNormal",
+  "subHeadingSmall",
+  "subHeading",
+  "bodyLarge",
+  "bodyNormal",
+  "bodySmall",
+  "body",
+] as const;
+
+export const TextTypesPossible = [
+  "primaryLight",
+  "primaryNormal",
+  "primaryBold",
+  "primary",
+  "secondaryLight",
+  "secondaryNormal",
+  "secondaryBold",
+  "secondary",
+  "successLight",
+  "successNormal",
+  "successBold",
+  "success",
+  "errorLight",
+  "errorNormal",
+  "errorBold",
+  "error",
+  "warningLight",
+  "warningNormal",
+  "warningBold",
+  "warning",
+  "light",
+  "normal",
+  "bold",
+] as const;
+
+export const VariantTypes: {
+  [key in (typeof VariantTypesPossible)[number]]: TextStyle;
+} = {
+  headingLarge: {
+    fontSize: 30,
+    fontWeight: "bold",
+    lineHeight: 36,
+    letterSpacing: 0,
+  },
+  headingNormal: {
+    fontSize: 26,
+    fontWeight: "bold",
+    lineHeight: 32,
+    letterSpacing: 0,
+  },
+  headingSmall: {
+    fontSize: 22,
+    fontWeight: "bold",
+    lineHeight: 28,
+    letterSpacing: 0,
+  },
+  heading: {
+    fontSize: 26,
+    fontWeight: "bold",
+    lineHeight: 32,
+    letterSpacing: 0,
+  },
+
+  subHeadingLarge: {
+    fontSize: 24,
+    fontWeight: "normal",
+    lineHeight: 28,
+    letterSpacing: 0,
+  },
+  subHeadingNormal: {
+    fontSize: 22,
+    fontWeight: "normal",
+    lineHeight: 26,
+    letterSpacing: 0,
+  },
+  subHeadingSmall: {
+    fontSize: 20,
+    fontWeight: "normal",
+    lineHeight: 24,
+    letterSpacing: 0,
+  },
+  subHeading: {
+    fontSize: 22,
+    fontWeight: "normal",
+    lineHeight: 26,
+    letterSpacing: 0,
+  },
+
+  bodyLarge: {
+    fontSize: 20,
+    fontWeight: "normal",
+    lineHeight: 26,
+    letterSpacing: 0,
+  },
+  bodyNormal: {
+    fontSize: 18,
+    fontWeight: "normal",
+    lineHeight: 22,
+    letterSpacing: 0,
+  },
+  bodySmall: {
+    fontSize: 16,
+    fontWeight: "normal",
+    lineHeight: 20,
+    letterSpacing: 0,
+  },
+  body: {
+    fontSize: 18,
+    fontWeight: "normal",
+    lineHeight: 22,
+    letterSpacing: 0,
+  },
+};
+
```

Comment:
@manojsai-bellamkonda  We already have a colors file from where we are accessing required color throughout the app. Check we can add your colors there and use from that place

Reply:
Sure Shashi. I will use the already defined colors from the colors file.